pxr/imaging/hdSt/codeGen.cpp

```diff
@@ -4611,6 +4718,8 @@ HdSt_CodeGen::_GenerateDrawingCoord(
                                   /*name=*/name,
                                   /*dataType=*/_tokens->_int);
         }
+
+        _PlumbInterstageElements();
```

Comment:
Turns out this `if (!_hasCS) { ... _PlumbInterstageElements(); }` block should be moved after the `static const std::vector<std::string> drawingCoordParams { ... }` block near the top of the function. Otherwise the code at line 4433 doesn't find the mapping. It's also missing the "vs_" prefix.